src/pages/Selector/index.jsx

```diff
@@ -38,3 +39,7 @@ const Selector = () => (
 );
 
 export default container(Selector);
+
+Selector.propTypes = {
+  employees: PT.arrayOf(PT.object).isRequired,
```

Comment:
What're you thoughts on shape rather than object? then set the used properties to .isRequired